cypress/utils/utils.ts

```diff
@@ -1814,6 +1814,21 @@ export function getCommandOutput(command: string): Cypress.Chainable<Cypress.Exe
     });
 }
 
+export function validateMtaVersionInCli(expectedMtaVersion: string): void {
+    const namespace = getNamespace();
+    const podName = `$(oc get pods -n${namespace}| grep ui|cut -d " " -f 1)`;
+    const command = `oc describe pod ${podName} -n${namespace}| grep -i version|awk '{print $2}'`;
+    getCommandOutput(command).then((output) => {
+        if (expectedMtaVersion === output.stdout) {
+            console.log("CLI version is as expected.");
```

Comment:
Why do we need a `console.log` here?

Can we just throw the error if  `expectedMtaVersion !== output.stdout`?

Reply:
Done